wandb/util.py

```diff
@@ -1944,8 +1944,8 @@ def get_core_path() -> str:
     bin_path = pathlib.Path(__file__).parent / "bin" / "wandb-core"
     if not bin_path.exists():
         raise WandbCoreNotAvailableError(
-            f"File not found: {bin_path}."
-            "Please contact support at support@wandb.com."
+            f"File not found: {bin_path}. "
+            "Please contact support at support@wandb.com. "
             f"Your platform is: {platform.platform()}."
         )
```

Comment:
nit: I'd put the spaces at the start of each line rather than at the end, so that they're easier to see and harder to forget:

```python
        raise WandbCoreNotAvailableError(
            f"File not found: {bin_path}."
            " Please contact support at support@wandb.com."
            f" Your platform is: {platform.platform()}."
        )
```

this way, if you remove a line, you don't have to remove a space from the end of the line above it